### DISCUSSION TITLE
使用 Astro 搭建博客系统 - CafeBlogs

### DISCUSSION THREAD

**Discussion by giscus** [CafeKiller/CafeKiller.github.io#1] [General]
# 使用 Astro 搭建博客系统 - CafeBlogs

A blog page built using the astro framework | 使用 astro 框架构建的博客页面

http://localhost:4321/MyBlog/blog/post240706

**Comment by CafeKiller** [comment]
配合 giscus 与 github discussions 使用，可以快速、低成本的搭建一个可用的评论系统
